api/src/pcapi/routes/backoffice/accounts/update_request_blueprint.py

```diff
@@ -234,12 +234,16 @@ def list_account_update_requests() -> utils.BackofficeResponse:
 
     autocomplete.prefill_bo_users_choices(form.last_instructor)
 
+    form_url = partial(url_for, ".list_account_update_requests", **form.raw_data)
+    date_last_update_sort_url = form_url(order="desc" if form.order.data == "asc" else "asc")
```

Comment:
```suggestion
    date_last_update_sort_url = form_url(order="asc" if form.order.data == "asc" else "desc")
```
c'est normal que tu aies inversé l'ordre par rapport au formulaire ?

Reply:
vous mùe faites douter, je reteste en local